odoo/sql_db.py

```diff
@@ -90,6 +90,17 @@ def clear_env(cr):
 
 sql_counter = 0
 
+QUERY_TO_CATCHS = [
+    b'SELECT "sale_order_line".id FROM "sale_order_line" LEFT JOIN "sale_order" AS "sale_order_line__order_id" ON ("sale_order_line"."order_id" = "sale_order_line__order_id"."id") WHERE (((((((TRUE AND "sale_order_line"."product_id" IS NOT NULL) AND ("sale_order_line"."order_id" IN',
+    b'SELECT "sale_order".id FROM "sale_order" WHERE ("sale_order"."state" in ("sale","done")) ORDER BY "sale_order"."id"',
```

Comment:
I'm not sure if the original query has not spaces between `,`
 - `"state" in ("sale","done")`

So, couldn't be catch

What about using a `str.replace` of spaces ` ` in order to avoid affecting because this?